packages/open-payments/src/client/outgoing-payment.test.ts

```diff
@@ -280,38 +281,58 @@ describe('outgoing-payment', (): void => {
   })
 
   describe('createOutgoingPayment', (): void => {
-    const quoteId = `${serverAddress}/quotes/${uuid()}`
+    const quoteId_ = `${serverAddress}/quotes/${uuid()}`
+    const incomingPaymentId = `${serverAddress}/incoming-payments/${uuid()}`
+    const debitAmount = {
+      value: '2500',
+      assetCode: 'USD',
+      assetScale: 2
+    }
 
     test.each`
-      metadata
-      ${{ description: 'Some description', externalRef: '#INV-1' }}
-      ${undefined}
-    `('creates outgoing payment', async ({ metadata }): Promise<void> => {
-      const outgoingPayment = mockOutgoingPayment({
+      quoteId      | incomingPaymentId    | debitAmount    | metadata                                                      | createSource
+      ${quoteId_}  | ${undefined}         | ${undefined}   | ${{ description: 'Some description', externalRef: '#INV-1' }} | ${'quote'}
+      ${quoteId_}  | ${undefined}         | ${undefined}   | ${undefined}                                                  | ${'quote'}
+      ${undefined} | ${incomingPaymentId} | ${debitAmount} | ${undefined}                                                  | ${'incoming payment'}
+    `(
+      'creates outgoing payment from $createSource',
```

Comment:
refactors our create outgoing payment test to not assume `quoteId`. Now either `quoteId` or `incomingPaymentId`/`debitAmount` are passed into the test as parameters.